modules/psap-driver-toolkit-pulling.adoc

```diff
@@ -24,11 +24,15 @@ The driver-toolkit image for the latest minor release will be tagged with the mi
 
 .Procedure
 
-. The image URL of the `driver-toolkit` corresponding to a certain release can be extracted from the release image using the `oc adm` command:
+. The image URL of the `driver-toolkit` corresponding to a certain release are extracted from the release image using the `oc adm` command:
```

Comment:
> Extract the image URL of the `driver-toolkit` corresponding to a certain release from the release image using the `oc adm` command:
??

Reply:
It sounded better in my head :man_shrugging: :joy:  I'll change it back!